conf/etc/iem_ha.json

```diff
@@ -0,0 +1,26 @@
+{
+    "<HOST1>": {
+        "lost": {
+            "event": "NodeFailure",
+            "IEM": "IEC:WS0080010001:The cluster has lost <UUID1> server; System is running in degraded mode."
```

Comment:
`<UUID1>`

Should we use srvnode-1/2 insted uuid
@ajaysrivas @indrajitzagade

Reply:
yes @ajay-paratmandali

Reply:
Done